This is not an autoconf build. acconfig.h will never exist.
--- inc/config.h.orig	2014-04-10 05:53:05.000000000 -0500
+++ inc/config.h	2018-11-05 10:26:43.000000000 -0600
@@ -11,7 +11,7 @@
 #ifndef __CONFIG_H__
 #define __CONFIG_H__
 
-#ifdef HAVE_CONFIG_H
+#if 0
 #include "acconfig.h"
 
 #else // not HAVE_CONFIG_H
